### DISCUSSION TITLE
Welcome to AI-Waifu-Vtuber Discussions!

### DISCUSSION THREAD

**Discussion by ardha27** [ardha27/AI-Waifu-Vtuber#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wjbwnd** [comment]
## Emotional expression

As an idea, you can link VTube studio and python code so that the model expresses emotions based on the generated text.
>There may be problems with animation for different models. Therefore, we need to optimize it somehow

**Reply by ardha27** [reply]
Yes i think it's a good idea. I have think about it, but i never try it

**Reply by wjbwnd** [reply]
when I was looking for repositories similar to yours, I found a similar idea from one person

here it is : https://github.com/HRNPH/AIwaifu

**Comment by wjbwnd** [comment]
And also you can also add a remote control to a neural network with buttons, for example, interrupt the generation of the current text and turn off the chat reading

**Reply by ardha27** [reply]
Do you have idea to do this? You can do push request

**Reply by wjbwnd** [reply]
I haven't thought about it yet, but if anything, I'll write
